docs/documentation/upgrading/topics/keycloak/changes-23_0_0.adoc

```diff
@@ -11,3 +11,12 @@ In these cases, it may be useful to disable adding the `iss` parameter to the au
 for the particular client in the {project_name} Admin console, in client details in the section with `OpenID Connect Compatibility Modes`,
 described in <<_compatibility_with_older_adapters>>. Dedicated `Exclude Issuer From Authentication Response` switch exists,
 which can be turned on to prevent adding the `iss` parameter to the authentication response.
+
+= Language files for themes default to UTF-8 encoding
+
+This release now follows the standard mechanisms of Java and later, which assumes resource bundle files to be encoded in UTF-8.
+
+Previous versions of Keycloak supported specifying the encoding in the first line with a comment like `# encoding: UTF-8`, which is no longer supported and is ignored.
+
+Message properties files for themes are now are read in UTF-8 encoding, with an automatic fallback to ISO-8859-1 encoding.
```

Comment:
Is it possible that someone uses some property files encoded in a different encoding than `UTF-8` or `ISO-8859-1`?

Reply:
Yes, it is possible that people used different encodings in the past, but no longer supported. 

I've added one more sentence to the migration guide: 

> If you are using a different encoding, convert the files to UTF-8.